### PR TITLE
Update ont-huawei-hn8010ts.md

### PR DESCRIPTION
**Pull request by stich86**
add useful information for XGSPON ONT

### CODE REVIEW

File: _ont_xgs/ont-huawei-hn8010ts.md [hunk 1]
```diff
@@ -11,20 +11,146 @@ parent: Huawei
 | ----------- | -------- |
```
Comment (simonebortolin) [inline _ont_xgs/ont-huawei-hn8010ts.md:11]:
```suggestion
| ----------- | --------------------------------------------------------------- |
| Vendor      | Huawei                                                          |
| Model       | HN8010Ts                                                        |
| Chipset     |                                                                 |
| Chipset     | HiSilicon SD5117                                                |
| Manufacter  |                                                                 |
| Flash       |                                                                 |
| RAM         |                                                                 |
| System      |                                                                 |
| Flash       | 128MB                                                           |
| RAM         | 32MB                                                            |
| System      | Dopra Linux                                                     |
| 10GBaseT    | Yes                                                             |
| Optics      |                                                                 |
| IP address  |                                                                 |
| Web Gui     |                                                                 |
| Optics      | SC/APC                                                          |
| IP address  | 192.168.100.1                                                   |
| Web Gui     | ✅ user: `telecomadmin`, password: `admintelecom`               |
| SSH         |                                                                 |
| Telnet      |                                                                 |
| Telnet      | After Enabling (via XML file) user: `root`, password: `adminHW` |
| Serial      |                                                                 |
| Form Factor | ONT                                                             |
```

and fix manual first line
```
|             |                                                                 |

```

File: _ont_xgs/ont-huawei-hn8010ts.md [hunk 2]
```diff
@@ -11,20 +11,146 @@ parent: Huawei
...
+It is possible that this won't work/has been fixed in more recent versions.
+
+Now that this is done, the modified configuration file can be uploaded via the web interface, the ONT will restart and telnet should be accessible on port 23.
+
+
+# General Settings and Useful Commands
+
+{% include alert.html content="All results need to be converted from HEX to ASCII. You can use our [Converter](/ascii-hex), taking care to add 0x in front and remove the spaces every 8 hex digits." alert="Warning" icon="svg-info" color="blue" %}
+
+## Getting the OLT vendor
```
Comment (simonebortolin) [inline _ont_xgs/ont-huawei-hn8010ts.md:54]: could you make a dedicated page to import since these instructions seem to be the same as the huawei hg8010h?

Comment (stich86) [reply _ont_xgs/ont-huawei-hn8010ts.md:54]: what do you mean? the procedure is exactly the same

Comment (stich86) [reply _ont_xgs/ont-huawei-hn8010ts.md]: i've create ont-huawei-useful-command.md file and put into _ont directory, please check if the reference URL is corret for XGS ONT and also for HG8010

**Review by simonebortolin — APPROVED** [review]

**Pull request merged**